api/v1/models/base.py

```diff
@@ -19,3 +19,33 @@
         Column('user_id', UUID(as_uuid=True), ForeignKey('users.id'), primary_key=True),
         Column('organization_id', UUID(as_uuid=True), ForeignKey('organizations.id'), primary_key=True)
         )
+
+class BaseModel():
+    """ This model creates helper methods for all models
+    """
+    def to_dict(self):
+        """ returns a dictionary representation of the instance
+        """
+        obj_dict = self.__dict__.copy()
+        del obj_dict["_sa_instance_state"]
+        obj_dict['id'] = str(self.id)
+        if self.created_at:
+            obj_dict["created_at"] = self.created_at.isoformat()
+        if self.updated_at:
+            obj_dict["updated_at"] = self.updated_at.isoformat()
+        return obj_dict
+
+    @classmethod
+    def get_all(cls):
+        """ returns all instance of the class in the db
+        """
+        from api.db.database import db
```

Comment:
Why?

Reply:
To prevent circular import error boss

Reply:
What is causing it this early in the project